docs/content/docs/components/dropdown.md

```
@@ -1,21 +1,43 @@
+
+<ComponentPreview name="Dropdown-03_TriggerPatterns" />
+
+## Content Customization
+
+<ComponentPreview name="Dropdown-04_ContentCustomization" />
+
+## User Menu
+
+<ComponentPreview name="Dropdown-04_UserMenu" />
+
+## Content Customization
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)